openid-federation-1_0.xml

```diff
@@ -5443,12 +5528,20 @@ Content-Type: application/json
 	<spanx style="verb">/.well-known/openid-federation</spanx>.
       </t>
       <t>
-        Federation Entities MUST make an Entity Configuration document
-        available at their configuration endpoint.
-        The sole exception to this
-        rule is for an RP that only does Explicit Registration.
+	Furthermore, any Entity Configuration including the Entity Type Identifier
+	<spanx style="verb">federation_entity</spanx> MUST be published
+	at its configuration endpoint.
+      </t>
+      <t>
+        While Leaf Federation Entities SHOULD make an Entity Configuration document
+        available at their configuration endpoints,
+	an exception to this requirement is that
+	an RP that only does Explicit Registration MAY omit doing so.
```

Comment:
Would like to  have this generalisation added.

Replace 5536-5539 with 

'An exception to this requirement are clients that uses a client registration process that results in the server having the clients Entity Configuration at the end of the registration process. They MAY omit doing so"

This would open the door for adding a method based on Aaron's dynamic client registation proposal